authentication/views.py

```diff
@@ -17,15 +17,20 @@ def well_known_openid_configuration(request):  # noqa: ARG001
     # See: https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfig
     # NOTE: this is intentionally incomplete because we don't fully support OpenID
     #       this was implemented solely for digital credentials
+
+    # In dev, we need to use host.docker.internal to reach the token & userinfo endpoint
+    token_base_url = settings.SITE_BASE_URL
+    if settings.ENVIRONMENT == "dev":
+        token_base_url = "http://host.docker.internal:9080"  # noqa: S105
```

Comment:
The value `http://host.docker.internal:9080` won't work across all development environments (including mine), can this instead be an environment variable that defaults to `SITE_BASE_URL` instead?